pilz_teleoperation/test/unittests/test_data/pose_file.py

```diff
@@ -0,0 +1,53 @@
+from geometry_msgs.msg import PoseStamped
+from geometry_msgs.msg import Point
+from geometry_msgs.msg import Pose
+from std_msgs.msg import Header
+from geometry_msgs.msg import Quaternion
+from rospy.rostime import Time
+
+start_pose_test = PoseStamped(
+    header=Header(
+        seq=0,
+        stamp=Time(
+            secs=12345,
+            nsecs=6789
+        ),
+        frame_id='world'
+    ),
+    pose=Pose(
+        position=Point(
+            x=7,
+            y=0.0,
+            z=0.0
+        ),
+        orientation=Quaternion(
+            x=0.0,
+            y=0.0,
+            z=0.0,
+            w=-1.0
+        )
+    )
+)
+goal_pose = PoseStamped(
+    header=Header(
+        seq=0,
+        stamp=Time(
+            secs=123,
+            nsecs=89
+        ),
+        frame_id='tcp'
```

Comment:
```suggestion
        frame_id='prbt_tcp'
```
to be displayed by the default robot model started with `roslaunch prbt_jog_arm_support prbt_jog_arm_sim.launch`

Reply:
this is test data, that is not prbt specific, and it is not supposed to be displayed in real